core/controlplane/config/templates/cluster.yaml

```diff
@@ -191,20 +191,6 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 #    # References subnets defined under the top-level `subnets` key by their names
 #    - name: ManagedPublicSubnet1
 #    - name: ManagedPublicSubnet2
-#  loadBalancer:
```

Comment:
Could you comment `CAUTION: Deprecated and will be removed in v0.9.7. Please use apiEndpoints[].loadBalancer instead` on this like we do for other deprecated keys?
so that our users won't lose track to new keys